xla/tests/constants_test.cc

```diff
@@ -121,6 +121,28 @@ TEST_F(ConstantsTest, OneCellF8e5m2) {
   ComputeAndCompareR1<float>(&builder, {2.0f}, {}, error_spec_);
 }
 
+TEST_F(ConstantsTest, OneCellF8e4m3) {
+  std::vector<tsl::float8_e4m3> constant = {tsl::float8_e4m3{2.0}};
+
+  XlaBuilder builder(TestName());
+  auto c = ConstantR1<tsl::float8_e4m3>(&builder, constant);
+  // F8 outputs are not yet supported so convert to F32
```

Comment:
This comment is actually no longer true. We should change the other tests with this comment as well. The test OneCellF8e5m2fnuz does have an FP8 output, so you can use that as an example in modifying this test.

If you want, you can also change the two existing tests with the comment to have FP8 outputs as well.

Reply:
Opened PR https://github.com/openxla/xla/pull/17182 - Parametrize ConstantsFloatTest OneCellFloat